api/diagnosis/diagnosis_routes_v2.py

```
@@ -0,0 +1,87 @@
+    logger.info(f"라즈베리 파이 UID: {device_uid} - 진단 결과: {is_drowsiness_drive}")
+    return JSONResponse(
+        status_code=200,
+        content={
+            "status": 200,
+            "success": True,
+            "isDrowsinessDrive": is_drowsiness_drive,
+            "detectionTime": detection_time
+        }
+    )
+
+async def get_frames_from_queue(device_uid: str):
```

Ruff (0.11.9)

Missing return type annotation for public function `get_frames_from_queue`

(ANN201)